packages/kit/src/constants.ts

```diff
@@ -2,19 +2,31 @@ export const CONTRACT_ADDRESSES = {
   testnet: {
```

Comment:
Are we doing emulator or pushing that once we figure out how to solve it better?